iron-overlay-manager.html

```diff
@@ -385,6 +386,7 @@
      * @param {!Element} overlay1
      * @param {!Element} overlay2
      * @return {boolean}
+     * @suppress {missingProperties}
```

Comment:
In the method we're already doing `/** @type {?} */ (overlay1)` for this reason, so we shouldn't need the `@suppress` here, right?